rktl_game/web/index.js

```diff
@@ -0,0 +1,79 @@
+// Connecting to ROS
+// -----------------
+
+var ros = new ROSLIB.Ros({
+    url: 'ws://r2d2.local:9090'
+});
+
+ros.on('connection', function () {
+    console.log('Connected to websocket server.');
+});
+
+ros.on('error', function (error) {
+    console.log('Error connecting to websocket server: ', error);
+});
+
+ros.on('close', function () {
+    console.log('Connection to websocket server closed.');
+});
+
+
+// Subscribe to the game manager node
+
+var statusListener = new ROSLIB.Topic({
+    ros: ros,
+    name: '/match_status',
+    messageType: 'rktl_msgs/MatchStatus'
+});
+
+// Creating the service objects called by the buttons
+
+var pause_srv = new ROSLIB.Service({
+    ros: ros,
+    name: 'pause_game',
+    serviceType: 'std_srvs/Empty'
+});
+
+var unpause_srv = new ROSLIB.Service({
+    ros: ros,
+    name: 'unpause_game',
+    serviceType: 'std_srvs/Empty'
+});
+
+var reset_srv = new ROSLIB.Service({
+    ros: ros,
+    name: 'reset_game',
+    serviceType: 'std_srvs/Empty'
+});
+
+// Game clock parameters
+
+var game_time_param = new ROSLIB.Param({
+    ros: ros,
+    name: 'game_length'
```

Comment:
This is the shared parameter? Should it be set in a config file? I don't see a default value nor a value set otherwise.